src/planscape/config/conditions.json

```diff
@@ -360,7 +377,8 @@
                   "data_year": "2021",
                   "reference_link": "https://data.fs.usda.gov/geodata/rastergateway/regional-resource-kit/docs/RRK_Metric_Dictionary_v2.pdf#page=16",
                   "min_value": 0,
-                  "max_value": 10
+                  "max_value": 10,
+                  "data_units": "Probability"
```

Comment:
probabilities are measured in %. why is the max 10?

Reply:
something for us to ask Pat and Nick! there's many metrics whose max we have in here do not match up with the `potential` max from the data dict